source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -44,28 +44,19 @@ workflow:
 #. Add the repository to `extract-translation-source-files.yml`_ in the `openedx-translations repo`_.
 
    Add a new entry under the ``python-translations`` section. For example for the XBlock located in
-   ``https://github.com/openedx/xblock-drag-and-drop-v2`` and has the XBlock Python 
-   module name ``drag_and_drop_v2`` it should have the following entry::
+   ``https://github.com/openedx/xblock-drag-and-drop-v2`` it should have the following entry::
```

Comment:
since I've highlighted code that isn't part of the diff I can't do a suggestion, so here's my manual one:
```
Add a new entry under the ``python-translations`` section. For example, the XBlock located at
``https://github.com/openedx/xblock-drag-and-drop-v2`` should have the following entry::
```

Reply:
done